Description: parser: Allow debugedit to work on apparmor_parser
 .
 https://launchpad.net/bugs/1561939
 .
 The debugedit program is unable to cope with the extra slash in the
 LIBAPPARMOR_SRC variable.
 .
 Signed-off-by: Tyler Hicks <tyhicks@canonical.com>
 Suggested-by: Christian Svensson <blue@cmd.nu>
Origin: commit, revision id: tyhicks@canonical.com-20160325152852-b0d1vaxrhtzhusv1
Author: Tyler Hicks <tyhicks@canonical.com>
Bug: https://launchpad.net/bugs/1561939
Last-Update: 2016-03-25
X-Bzr-Revision-Id: tyhicks@canonical.com-20160325152852-b0d1vaxrhtzhusv1

=== modified file 'parser/Makefile'
--- old/parser/Makefile	2015-06-24 23:18:04 +0000
+++ new/parser/Makefile	2016-03-25 15:28:52 +0000
@@ -95,7 +95,7 @@
   INCLUDE_APPARMOR =
   APPARMOR_H =
 else
-  LIBAPPARMOR_SRC = ../libraries/libapparmor/
+  LIBAPPARMOR_SRC = ../libraries/libapparmor
   LOCAL_LIBAPPARMOR_INCLUDE = $(LIBAPPARMOR_SRC)/include
   LOCAL_LIBAPPARMOR_LDPATH = $(LIBAPPARMOR_SRC)/src/.libs
 

